### PR TITLE
Ports Cool Stuff from Bastion of Hestia/Vesta, Adds Eye Layer-Switching

### PR DESCRIPTION
**Pull request by KasparoVy**
Just going all out on some new content, this stuff is also pretty dope.

Everything has been tested prior to opening this PR and another pass of testing has been completed successfully after the last commit.

**Overall Changes:**
- Fully implements DNA-integrated secondary & tertiary colouration for ears, wings and tails.
- Refactors update_icons_vr.dm into the one file within `/code/modules/vore/appearance`
- Adds some new ears, wings, tails and markings from Bastion of Hestia/Vesta.
- Adds a verb you can use to render eyes above markings instead of below.
- Adds an 'eyes' option to the Protogen monitor selection. They blink!

------------------

**Ears:**
- Plain, thick & curly synth horns.

**Wings:**
- Multiple new harpy wing variations (incl. archeopteryx & bat)
- Neckfur

**Tails:**
- Eastern dragon (animated for wagging)
- Synth lizard tail + wagging version
- Synth plug tail
- Segmented tail + wagging version
- Synth segmented tail (has lights), animated
- A couple fox tails
- Fennec tail
- A few lizard tails (looks like Citadel/TG version)
- Xenomorph tail (looks like Citadel version - properly badass)
- A properly colourable xeno tail (multiplicative blending)

**Markings:**
- Some more Vox markings
- Unathi cobra hood as marking
- Unathi frill & horns as markings

------------------
_No Synths were harmed in the making of this PR._
virgo ss13 redsynth 20201121virgo ss13 blusynth 20201121

### CODE REVIEW

**Comment by KasparoVy** [comment]
Another commit coming with some improvements

**Comment by KasparoVy** [comment]
Don't mind the out of sync synths, they're doing their best OK?

Ready.

**Comment by KasparoVy** [comment]
Don't mind the out of sync synths, they're doing their best OK?

Ready. Even that eye rendering verb works.
@Novacat This may also be best to include in the hotfix prior to https://github.com/VOREStation/VOREStation/pull/9327

File: code/modules/organs/organ_icon.dm [hunk 1]
```diff
@@ -79,8 +81,14 @@ var/global/list/limb_icon_cache = list()
 		//We have weird other-sorts of eyes (as we're not supposed to have eye organ, but we have HAS_EYE_COLOR species)
 		else
 			eyes_icon.Blend(rgb(owner.r_eyes, owner.g_eyes, owner.b_eyes), ICON_ADD)
-		add_overlay(eyes_icon)
-		mob_icon.Blend(eyes_icon, ICON_OVERLAY)
+
+		//VOREStation edit -- allow rendering of eyes over markings.
```
Comment (Novacat) [inline code/modules/organs/organ_icon.dm:85]: Would this mess with the heterochromia marking?

Comment (KasparoVy) [reply code/modules/organs/organ_icon.dm:85]: Yes, but if you want to render your eyes above markings that is a caveat that must be understood-- eyes/monitor styles will render above all markings

However if your 'eye'/monitor sprite is largely transparent, or if the eye sprite only has pixels for the one eye to begin with, then the heterochromia will still render-- it won't be clipped out by the transparency of the eye sprite above

**Comment by KasparoVy** [comment]
NOICE

**Pull request merged**